src/impls.rs

```diff
@@ -173,16 +173,29 @@ mod test {
     #[test]
     #[cfg(feature = "with-chrono-0_4")]
     fn test_tsrange_params() {
-        let low = Utc.timestamp(0, 0);
+        let d = NaiveDate::from_ymd_opt(2015, 6, 3).unwrap();
+        let t = NaiveTime::from_hms_milli_opt(12, 34, 56, 789).unwrap();
+
+        let low = NaiveDateTime::new(d, t);
         let high = low + Duration::days(10);
-        test_range!("TSRANGE", NaiveDateTime, low.naive_utc(), "1970-01-01", high.naive_utc(), "1970-01-11");
+        test_range!("TSRANGE", NaiveDateTime, low, "2015-06-03T12:34:56.789", high, "2015-06-13T12:34:56.789");
     }
 
     #[test]
     #[cfg(feature = "with-chrono-0_4")]
     fn test_tstzrange_params() {
-        let low = Utc.timestamp(0, 0);
+        let low = Utc.with_ymd_and_hms(2014, 7, 8, 9, 10, 11).unwrap();
+        let high = low + Duration::days(10);
+        test_range!("TSTZRANGE", DateTime<_>, low, "2014-07-08T09:10:11Z", high, "2014-07-18T09:10:11Z");
+    }
+
+
+    #[test]
+    #[cfg(feature = "with-chrono-0_4")]
+    #[ignore = "Exclusive starts will be converted to next-day + inclusive"]
```

Comment:
I believe this is exactly what the `normalize` method is supposed to handle.

Reply:
That's good to know, thanks! I'll have another look at it and see if I can adapt my code :-)